test/application_test.rb

```diff
@@ -6,8 +6,8 @@
   end
 
   it 'instantiate new configuration object when inherited' do
-    backend_app  = Backend::Application.new
-    frontend_app = Frontend::Application.new
+    backend_app  = Back::Application.new
```

Comment:
Well, this is not related with jRuby, indeed. But with this rename we're able to standalone run this spec with `ruby -I test test/application_test.rb`.

`Backend::Application` and `Frontend::Application` are defined in a deep microservices fixture while `Back::Application` and `Front::Application` are defined in the `fixture.rb` itself.

And more importante: today they may fail if this test runs before `test/fixtures/microservices` spec loads.